lib/chef/knife/winrm_knife_base.rb

```diff
@@ -46,6 +54,8 @@ def validate_options!
           def configure_session
             validate_options!
             resolve_session_options
+            load_windows_specific_gems if @session_opts[:transport] == :sspinegotiate
+            warn_no_ssl_peer_verification if @session_opts[:no_ssl_peer_verification]
```

Comment:
note this line forces a warning even when NOT using SSL. Rather than addressing here, I'm gonna wait for this to go through and submit separately.